cmake/CMakeLists.txt

```diff
@@ -809,7 +809,10 @@ if (onnxruntime_ENABLE_TRAINING)
       set(HOROVOD_ROOT ${PROJECT_SOURCE_DIR}/external/horovod)
       set(HOROVOD_INCLUDE_DIRS  "${HOROVOD_ROOT}/horovod/common" ${MPI_CXX_INCLUDE_PATH})
       add_subdirectory(horovod EXCLUDE_FROM_ALL)
-      list(APPEND onnxruntime_EXTERNAL_LIBRARIES horovod  /usr/lib/x86_64-linux-gnu/libnccl.so ${MPI_CXX_LIBRARIES} ${MPI_CXX_LINK_FLAGS})
+      message("Use gloo from submodule")
```

Comment:
This may be more appropriate as a comment instead of output. Could also be more specific, like use external/horovod/third_party/gloo/cmake/Modules/Findnccl.cmake or something.